tmc-langs-rust/src/main/java/fi/helsinki/cs/tmc/langs/rust/CargoPlugin.java

```diff
@@ -61,7 +63,7 @@ protected StudentFilePolicy getStudentFilePolicy(Path projectPath) {
     public ValidationResult checkCodeStyle(Path path) {
         if (run(new String[]{"cargo", "clean"}, path).isPresent()) {
             String[] command = {"cargo", "rustc", "--", "--forbid", "warnings"};
-            log.info("Building for lints with command {0}", Arrays.deepToString(command));
+            log.info("Building for lints with command {}", Arrays.deepToString(command));
```

Comment:
Nit: I'm wondering if there is need for the `deepToString` though I don't know whether it has any efficiency implications... 
I'm not saying we need to change it, I just keep on wondering about this.